pkg/plugin/aggregation/aggregator_test.go

```diff
@@ -125,6 +125,9 @@ func TestAggregation_tarfile(t *testing.T) {
 			}
 		} else {
 			t.Errorf("AggregationServer didn't record a result for e2e tests. Got: %+v", agg.Results)
+			for k, v := range agg.Results {
+				t.Logf("Result %q: %+v\n", k, v)
```

Comment:
The set of results is a map so you get just the `key: <memory address>` when printing the map above. This makes it clear what each result is if you get an error (which I did while making these changes)